ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v1/fileinput/clientlibs/site/js/fileinputview.js

```diff
@@ -72,15 +72,15 @@
 
         updateValue(value) {
             if (this.widgetObject == null) {
-                this.widgetObject = new FileInputWidget(this.getWidget(), this.getFileListDiv(), this._model)
+                this.widgetObject = new FileInputWidget(this.getWidget(), this.getFileListDiv(), this._model, FormView.LanguageUtils)
             }
             this.widgetObject.setValue(value);
         }
 
         setModel(model) {
             super.setModel(model);
             if (this.widgetObject == null) {
-                this.widgetObject = new FileInputWidget(this.getWidget(), this.getFileListDiv(), this._model)
+                this.widgetObject = new FileInputWidget(this.getWidget(), this.getFileListDiv(), this._model, FormView.LanguageUtils)
```

Comment:
FormView is already at global scope. we need not pass it. Any specific reason here?